fe1-web/src/features/wallet/screens/__tests__/WalletError.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { expect } from '@jest/globals';
+import { render } from '@testing-library/react-native';
+import React from 'react';
+
+import MockNavigator from '__tests__/components/MockNavigator';
+
+import { WalletError } from '../index';
+
+const mockSeed = 'one two three';
+
+jest.mock('../../objects/Seed', () => {
```

Comment:
It might be better to use an absolute path for consistency with other tests. 
By the way, I think it's nicer if blocks are separated by an empty line.